suse_openstack_cloud

```diff
@@ -134,6 +136,8 @@ _capture_filter() {
         # "bmc_password": "s3kr1t",
         # "machine-install": { "password": "s3kr1t" },
         -e 's/\("\([a-z_]\+\)\?\(passw\(or\)\?d\|token\)":\? \+"\+\)[^"]\+"/\1'"$CENSORED"'"/gi'
+        # 'password': 's3kr1t',
+        -e "s/\('\([a-z_]\+\)\?\(passw\(or\)\?d\|token\)':\? \+'\+\)[^']\+'/\1$CENSORED'/gi"
```

Comment:
I guess that is expected to fail in case of escaped strings, like `'x\'x'`. The other regex have the same problem, IIUC.